### PR TITLE
Use hash maps from fxhash instead of fnv

### PR DESCRIPTION
**Pull request by andidr**
Replace the hash maps from `fnv` with implementations from `fxhash`, resulting in a ~30% decrease of execution time of the `descent` bench.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,10 +12,6 @@ repository = "https://github.com/ulysseB/telamon/"
 name = "descent"
 harness = false
 
-[[bench]]
```
Comment (Elarnon) [inline Cargo.toml]: Revert this

Comment (andidr) [reply Cargo.toml]: That shouldn't have been committed. Fixed in the update.

**Comment by andidr** [comment]
bors r=Elarnon

**Comment by andidr** [comment]
bors retry

**Pull request merged**